### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by shubhi-raft**
### This pull request changes...
replaced PR checklist w/ qasp review checklist

## FUNCTIONALITY
*Describe the functionality*
*Addresses Issue #*

## TO TEST

*List the steps to test the PR*

### This pull request is ready to merge when...

- [ ] Meets acceptance criteria for the issue 
- [ ] Code is reviewed by someone other than the original author
- [ ] Code is tested 
- [ ] Experience is approved by UX  
- [ ] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [ ] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [ ] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,26 +1,69 @@
...
++ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
+    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
+
+### Deliverable 5: Deployed
+
+> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*
+
+> *Acceptable Quality Level: Successful build with a single command*
+
++ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?
```
Comment (jtwillis92) [inline .github/pull_request_template.md:50]: @carltonsmith correct me if I'm wrong but as I understand it nothing currently gets deployed to the development instance until it gets merged in by HHS. If so, we may not be able to complete this particular step during Raft review.

That may change once the staging site is set up, and would probably be better if we could deploy to the dev environment prior to submitting for QASP review so we can isolate any issues ahead of time and prevent needing to rollback the HHS repo.

**Review by jtwillis92 — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (carltonsmith) [reply .github/pull_request_template.md:50]: @jtwillis92 Yes, this process changed from when we originally developed this checklist, so you are right, this doesn't happen until after it is merged to HHS. We do have a way of launching an automated deployment when we merge with `raft-review` but I need to find a cleaner way of doing that, as we can often get things in there that aren't part of what we are testing.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,26 +1,69 @@
-### This pull request changes...
```
[the remark this answers is not in the document]
Comment (carltonsmith) [reply .github/pull_request_template.md:1]: I also think that we should have a section to post the ACs from the issue and a follow up section to explain how this PR addresses the ACs

**Review by carltonsmith — CHANGES_REQUESTED** [review]
Thanks for getting this together! Just need a couple more things for it

**Comment by shubhi-raft** [comment]
@carltonsmith @jtwillis92 feel free to update anything else that does not make sense and push this.

File: .github/pull_request_template.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jtwillis92) [reply .github/pull_request_template.md:1]: For that last suggestion, that's the intention of the `Deliverable 1` section of the QASP checklist right?

**Review by jtwillis92 — APPROVED** [review]
This looks good to me!

**Review by carltonsmith — APPROVED** [review]
LGTM!

**Pull request merged**